sdk/nodejs/x/automation/workspace.ts

```diff
@@ -171,16 +171,18 @@ export interface Workspace {
      *
      * @param name the name of the plugin.
      * @param version the version of the plugin e.g. "v1.0.0".
+     * @param kind the kind of plugin e.g. "resource"
      */
-    installPlugin(name: string, version: string): Promise<void>;
+    installPlugin(name: string, version: string, kind?: string): Promise<void>;
```

Comment:
Why not strongly type as `PluginKind`?

Reply:
Ah sorry, looks like I missed this! I will follow up on this in my next changeset.